walkthroughs/week-5-tdd/intro/src/test/java/com/google/sps/GreeterTest.java

```diff
@@ -30,4 +30,24 @@ public void testGreeting() {
 
     Assert.assertEquals("Hello Ada", greeting);
   }
+
+  @Test
+  public void testGreetingTrimsWhitespace() {
+  Greeter greeter = new Greeter();
+
+  String greeting = greeter.greet("   Ada   ");
+
+  // Whitespace should be trimmed
+  Assert.assertEquals("Hello Ada", greeting);
```

Comment:
These lines should be indented, and not line up with the `public void testGreeting...`